### ISSUE TITLE
Hi Luke! I have created a dual system using Coolbooter. How do I mount the secondary system?

### ISSUE THREAD

**Issue by yoyo2306** [LukeZGD/Legacy-iOS-Kit#622]

**Comment by LukeZGD** [comment]
This doesnt really have anything to do with Legacy ios kit but ig i can help you. Using ssh or terminal, use the following commands:

`mount_hfs /dev/disk0s1s3 <mount folder>` for root

`mount_hfs /dev/disk0s1s4 <mount folder>` for data

replace <mount folder> with the path where you want to mount the fs

**Comment by yoyo2306** [comment]
The operation you mentioned above did not work, and the secondary system used by coolbooer could not be mounted.
My device is iPhone5. Because it does not have SHSH blob, it cannot be downgraded to ios6. Therefore, I used coolbooter to create a dual system. After entering the secondary system ios6, I found that the device has an activation lock, which makes the ios6 system unable to activate. Then I used Legacy-iOS-Kit to extract the activation file. I wanted to restore the activation file to the secondary system ios6 by mounting the device, but I found that Legacy-iOS-Kit only mounted the main system, so I could not restore the activation file to the secondary system ios6 for activation. How can I mount the secondary system ios6? It would be better to be more specific. Thank you
